contracts/contracts/ramps/wise/interfaces/ITLSData.sol

```diff
@@ -0,0 +1,11 @@
+//SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.18;
+
+interface ITLSData {
+    struct TLSParams {
+        address verifier;
```

Comment:
I'm not gonna split them out, they all make sense together as requirements to validate any one TLS proof. You need all three of those values otherwise you can't validate.